lib/svn2git/migration.rb

```diff
@@ -154,6 +154,7 @@ def parse(args)
 
     def self.escape_quotes(str)
       str.gsub(/'|"/) { |c| "\\#{c}" }
+         .gsub(/\$/)  { |c| '\\$' }
```

Comment:
Possibly.  I am not a ruby guy, but it looks good.  However, we are not just passing this to the shell, we are passing it to the shell and putting it in double quotes.  I think if we used shellwords.escape we might also want to change all the calling instances to *not* be wrapped in double quotes.  Not sure.